lib/workload/stateless/postgres_manager/deploy/postgres-manager-stack.ts

```diff
@@ -16,13 +16,13 @@ export type PostgresManagerConfig = {
   clusterResourceIdParameterName: string;
 };
 
-export type PostgresManagerStackProps = PostgresManagerConfig & {
+export type PostgresManagerProps = PostgresManagerConfig & {
   vpc: ec2.IVpc;
   lambdaSecurityGroup: ec2.ISecurityGroup;
 };
 
-export class PostgresManager extends Construct {
-  constructor(scope: Construct, id: string, props: PostgresManagerStackProps) {
+export class PostgresManagerStack extends Stack {
```

Comment:
@mmalenic am thinking of making microservice to stack instead of construct. One of the thought is that we could have the control for each microservice stack (e.g. we could deploy `yarn cdk-stateless-pipeline deploy ${stateless-microservice-stack}`  instead of the whole stateless stack). Do you have any thoughts or perhaps objection on this?

Reply:
No objections, I think that's a good idea. What would be the process of deploying individual microservice stacks? Would there be a `individual_stacks.ts` file in `bin` that lists all the microservices, or is there a way deploy stack that are nested within other stacks?

Reply:
Hmmm, perhaps adding in the stack construct at the `bin/orcabus.ts` could be it and `yarn cdk-orcabus ls`  could list all the stacks? I usually deploy from `yarn cdk-stateless-pipeline deploy ${stateless-microservice-stack}` as there might be some namespace clashes (if deployed the same stack in the same account). Not sure what is the best pattern/approach yet thou.